src/mapgen.cpp

```diff
@@ -386,6 +386,8 @@ MgStoneType MapgenBasic::generateBiomes()
 	u32 index = 0;
 	MgStoneType stone_type = MGSTONE_STONE;
 
+	noise_filler_depth->perlinMap2D(node_min.X, node_min.Z);
```

Comment:
It's debatable.  Really biomegen shouldn't be concerned with node placement, but instead only deciding which biomes occur at which coordinates, IMHO.

Biomes are supposed to provide information to the mapgen about the type of area that exists at that point; what mapgens do with that information is up to them.